### PR TITLE
Pull from cloudwatch

### PR DESCRIPTION
**Pull request by JasonTheMain**
## Changes proposed in this pull request:
- Add in job for cloudwatch with blobs for cloudwatch plugin
-
-

## Security considerations

New job which uses AWS and has access to cloudwatch

### CODE REVIEW

File: jobs/ingestor_cloudwatch/templates/bin/ingestor_cloudwatch [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+# # append deployment dictionary files
+# <% p('logstash_parser.deployment_dictionary').each do |dictionary_path| %>
+#   cat "<%= dictionary_path %>" >> ${JOB_DIR}/config/deployment_lookup.yml
+# <% end %>
+
+# echo "filter {" >> ${JOB_DIR}/config/logstash.conf
+
+# cat ${JOB_DIR}/config/filters_pre.conf >> ${JOB_DIR}/config/logstash.conf
+# cat /var/vcap/packages/base-logstash-filters/logstash-filters-default.conf >> ${JOB_DIR}/config/logstash.conf
```
Comment (markdboyd) [inline jobs/ingestor_cloudwatch/templates/bin/ingestor_cloudwatch:78]: We can probably delete this

File: jobs/ingestor_cloudwatch/templates/bin/ingestor_cloudwatch [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+# cat ${JOB_DIR}/config/filters_override.conf >> ${JOB_DIR}/config/logstash.conf
+
+# cat ${JOB_DIR}/config/filters_post.conf >> ${JOB_DIR}/config/logstash.conf
+# <% if p('logstash_parser.enable_json_filter') %>
+#   cat /var/vcap/packages/base-logstash-filters/if_it_looks_like_json.conf >> ${JOB_DIR}/config/logstash.conf
+# <% end %>
+# cat /var/vcap/packages/base-logstash-filters/timecop.conf >> ${JOB_DIR}/config/logstash.conf
+# cat /var/vcap/packages/base-logstash-filters/deployment.conf >> ${JOB_DIR}/config/logstash.conf
+
+# echo "} #close filters" >> ${JOB_DIR}/config/logstash.conf
```
Comment (markdboyd) [inline jobs/ingestor_cloudwatch/templates/bin/ingestor_cloudwatch:99]: And all this

File: jobs/ingestor_cloudwatch/spec [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+    default: false
+  logstash_parser.message_max_size:
+    description: "Maximum log message length.  Anything larger is truncated (TODO: move this to ingestor?)"
+    default: 1048576
+  logstash_parser.filters:
+    description: "The configuration to embed into the logstash filters section. Can either be a set of parsing rules as a string or a list of hashes in the form of [{name: path_to_parsing_rules.conf}]"
+    default: ''
+  logstash_parser.deployment_dictionary:
+    description: "A list of files concatenated into one deployment dictionary file. Each file contains a hash of job name-deployment name keypairs for @source.deployment lookup."
+    default: [ "/var/vcap/packages/base-logstash-filters/deployment_lookup.yml" ]
```
Comment (markdboyd) [inline jobs/ingestor_cloudwatch/spec:146]: We're not using these, so probably we should delete them

File: jobs/ingestor_cloudwatch/templates/config/input_and_output.conf.erb [hunk 4]
```diff
@@ -0,0 +1,59 @@
+input
+{
+  cloudwatch_logs
+  {
+    log_group_prefix => true
+    log_group => ["/aws/rds/instance/"]
+    region => "us-gov-west-1"
```
Comment (markdboyd) [inline jobs/ingestor_cloudwatch/templates/config/input_and_output.conf.erb:7]: Should add a job spec variable for this

File: jobs/ingestor_cloudwatch/spec (hunk 3 above)
Comment (JasonTheMain) [reply jobs/ingestor_cloudwatch/spec]: Should be reflected now

**Review by markdboyd — APPROVED** [review]

**Pull request merged**